### ISSUE TITLE
Common Crawl

### ISSUE THREAD

**Issue by voidfiles** [rodricios/crawl-to-the-future#1]
This may not go back as far as you would like, but the data might be helpful.

http://commoncrawl.org

**Comment by rodricios** [comment]
Hey @voidfiles, thanks for the suggestion. I ended up looking at commoncrawl's dataset, but unfortunately their pages, [according to this thread, start at year 2008](https://groups.google.com/d/msg/common-crawl/ZlcqGLMbfoY/aD1vWq6vZhoJ). 

But thanks for taking the time to send the link!

In any case, the script I wrote, while I cringe each time I have to look at it, has proven to be for the most part [sort of] effective. 

The only complaint I have is that most of the domains I'm crawling had (may still have) exclusions in their robots.txt for crawlers on every page past the first forward link; this makes getting "bad" pages - like the landing pages of those domains - occur often and downloading sites w/ "content" occur less

While it's a small nuisance for me/the project, in the grand scheme of things it's kind of sad that those pages weren't archived in their original form and in the public domain. The closest thing you'll find is probably an "archived" version that the newspapers kept as pure text that gets inserted somewhere in the site's current template.
